packages/contracts/deploy/011_LossRatioHealthCheck.deploy.ts

```diff
@@ -1,31 +1,23 @@
-import { BlockchainType } from "../hardhat.config";
-import {
-  DeployConfig,
-  BaseDeploymentService,
-  BaseInitArgs,
-  wrap,
-} from "@eonian/upgradeable";
+import type { BaseInitArgs, DeployConfig } from '@eonian/upgradeable'
+import { BaseDeploymentService, wrap } from '@eonian/upgradeable'
+import { BlockchainType } from '../hardhat.config'
 
 /**
  * Deploy LossRationHealthCheck.
  */
 export const config: DeployConfig = {
-  contract: "LossRatioHealthCheck",
-  chains: [
-    BlockchainType.Mainnet,
-    BlockchainType.Testnet,
-    BlockchainType.Local,
-  ],
-  tags: ["Default"],
-};
+  contract: 'LossRatioHealthCheck',
+  chains: [BlockchainType.Mainnet, BlockchainType.Testnet, BlockchainType.Local],
+  tags: ['Default'],
+}
 
 export class LossRatioHealthCheckDeployment extends BaseDeploymentService {
   // eslint-disable-next-line no-empty-pattern
-  async onResolveInitArgs({}: BaseInitArgs): Promise<Array<any>> {
-    return [
+  onResolveInitArgs({}: BaseInitArgs): Promise<Array<any>> {
+    return Promise.resolve([
       500, // shutdownLossRatio
-    ];
+    ])
```

Comment:
Why this change?

Reply:
it restricts "async" without using "await"